dspace/config/dspace.cfg

```diff
@@ -1316,7 +1316,9 @@ websvc.opensearch.enable = true
 # url used in service document
 websvc.opensearch.svccontext = opensearch
 # context for html request URLs - change only for non-standard servlet mapping
-websvc.opensearch.uicontext = simple-search
+websvc.opensearch.uicontext = search
```

Comment:
This change appears to imply you expect `websvc.opensearch.uicontext` to be relative to the `dspace.ui.url`.  Unfortunately, though, currently in the codebase it's relative to `dspace.server.url`:  https://github.com/DSpace/DSpace/blob/main/dspace-api/src/main/java/org/dspace/app/util/OpenSearchServiceImpl.java#L98-L101

Maybe there's another bug in the codebase, or I'm misunderstanding this change?

Reply:
you are right, I fixed the code to look to the right configuration (as it is the **UI**context)